tests/ui/issues/issue-109681.stderr

```diff
@@ -0,0 +1,3 @@
+'common' global must have a zero initializer!
+ptr @_ZN12issue_1096815TEST317h0f138c604623c157E
```

Comment:
The symbol name includes a hash suffix which includes information that will change from one version of Rust to another. That means the next time we increment the Rust minor version, this hash will change and the test will fail since the exact symbol name will be slightly different.

I'd suggest marking this symbol as [`#[no_mangle]`](https://doc.rust-lang.org/reference/abi.html#the-no_mangle-attribute) in the test so that we get exactly the same name every time the test runs.